app/scripts/CenterTrack.js

```diff
@@ -61,7 +61,7 @@ class CenterTrack extends React.Component {
         this.rangeSelectionTriggeredXYEnd = false;
       }
       return this.state !== nextState;
-    } else if (this.props.rangeSelection !== nextProps.rangeSelection) {
+    } if (this.props.rangeSelection !== nextProps.rangeSelection) {
```

Comment:
This looks strange to me. Maybe put the new if on a new line?

Reply:
will do.